### PR TITLE
style: enhance FAQ container styling and adjust layout for customer s…

### PR DESCRIPTION
**Pull request by sharpshooter90**
fixed the button casing
faq layout shift issue due to parent flex

#### Related Issues / PR's
https://github.com/SigNoz/engineering-pod/issues/1806

#### Before behaviour
https://www.loom.com/share/1befb0e7104b4d0fb7167001f820afae?sid=a2162726-d327-4c52-bb5f-4aa3397b9f3f

#### Screenshots after
image
image
image

### CODE REVIEW

File: frontend/src/pages/WorkspaceLocked/WorkspaceLocked.tsx [hunk 1]
```diff
@@ -187,8 +187,26 @@ export default function WorkspaceBlocked(): JSX.Element {
 			children: (
 				<Row gutter={[24, 16]} justify="center">
 					{/* #FIXME: please suggest if there is any better way to loop in different columns to get the masonry layout */}
-					<Col span={10}>{renderCustomerStories((index) => index % 2 === 0)}</Col>
-					<Col span={10}>{renderCustomerStories((index) => index % 2 !== 0)}</Col>
+					<Col
+						span={10}
+						style={{
+							display: 'flex',
+							flexDirection: 'column',
```
Comment (vikrantgupta25) [inline frontend/src/pages/WorkspaceLocked/WorkspaceLocked.tsx:194]: can you move these to css files and add a classname here. we should not add inline styles

Comment (sharpshooter90) [reply frontend/src/pages/WorkspaceLocked/WorkspaceLocked.tsx:194]: done

File: frontend/src/pages/WorkspaceLocked/WorkspaceLocked.tsx [hunk 2]
```diff
@@ -187,8 +187,26 @@ export default function WorkspaceBlocked(): JSX.Element {
...
+							alignItems: 'flex-end',
+						}}
+					>
+						{renderCustomerStories((index) => index % 2 === 0)}
+					</Col>
+					<Col
+						span={10}
+						style={{
+							display: 'flex',
+							flexDirection: 'column',
```
[the remark this answers is not in the document]
Comment (sharpshooter90) [reply frontend/src/pages/WorkspaceLocked/WorkspaceLocked.tsx:204]: done

**Comment by YounixM** [comment]
@sharpshooter90 : Please add before and after screenshots.

**Review by vikrantgupta25 — APPROVED** [review]

**Review by rahulkeswani101 — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Pull request merged**